package/tektelic/data/etc/helium_gateway/settings.toml

```diff
@@ -0,0 +1,90 @@
+## gateway-rs default configuration file
+##
+## NOTE: Do _NOT_ change values in this file. 
+##   Override settings in settings.toml by putting the sections and
+##   settings that need to be changed there.  
+
+## Keypair is a string that supports specifying different locations for 
+## the secrets. The default one is file based one since all devices 
+## can support it.
+##
+## Override the setting by putting a keypair line in the 
+## local settings.toml. Changing things in this file will be overwritten
+## by upgrades. 
+##
+## File: 
+#keypair = "/etc/helium_gateway/gateway_key.bin"
+## ECC608 based
+keypair = "ecc://ttyO1:0&slot=0"
+listen = "127.0.0.1:1680"
+region = "US915"
+
+[log]
+method = "syslog"
+level = "info"
+timestamp = false
+
+[update]
+# Disable update checking
+enabled = false
+# OVERRIDE: Platform to use as part of update check. This MUST be overriden by a
+# local settings file
+platform = "unknown"
```

Comment:
Please remove from here on downwards.. default.toml already includes all these settings so there is no need to override them